desktop/cmp/pinpad/impl/PinPad.ts

```diff
@@ -22,7 +23,7 @@ import './PinPad.scss';
  */
 export function pinPadImpl({model, testId}, ref) {
     return frame({
-        ref: composeRefs(model.ref, ref),
+        ref: composeRefs(model.ref as ForwardedRef<HTMLDivElement>, ref),
```

Comment:
Curious why the cast here rather than specifying the type in the model

Reply:
I added props TS to PinPad in this [commit](https://github.com/xh/hoist-react/pull/3677/commits/9a7f3c8d72a92d869fa1d2354e4ddf5b14ffa0d0), so this all changed.